app/views/photos/index.html.erb

```diff
@@ -0,0 +1,5 @@
+<% @photos.each do |photo| %>
+  <div><img src="<%= photo.image_path %>" alt=""></div>
```

Comment:
we made the changes Thomas wanted, tests got green and it worked on the local host